packages/editor/src/importMeta.ts

```diff
@@ -0,0 +1,4 @@
+// Used for getting around the issue of using import meta in jest. This is a workaround to allow us to mock the import.meta object in jest tests.
+// See LicenseManager.test.ts for an example of how to mock this.
+const { MODE: IMPORT_META_ENV } = import.meta && (import.meta as any).env
```

Comment:
hm my Vite-fu isn't the best - i wonder if @SomeHats or @ds300 would have a better idea here 🤔

Reply:
I've simplified this a bit, but still using the same approach.
https://github.com/tldraw/tldraw/pull/4021/files#diff-15f06a16b6271f950e2caf09ed4d5f1a6df7becac09a69b4ea825637a3fabe2fR3

Reply:
hm, this seems to break other tests...